src/solvers/dgsem_tree/dg_3d_parabolic.jl

```diff
@@ -869,7 +873,8 @@ end
 # Calculate the gradient of the transformed variables
 function calc_gradient!(gradients, u_transformed, t,
                         mesh::TreeMesh{3}, equations_parabolic,
-                        boundary_conditions_parabolic, dg::DG, cache, cache_parabolic)
+                        boundary_conditions_parabolic, dg::DG, parabolic_scheme, cache,
+                        cache_parabolic)
```

Comment:
```suggestion
                        boundary_conditions_parabolic, dg::DG, parabolic_scheme, 
                        cache, cache_parabolic)
```

Semantic formatting?